test/test_bank.rb

```diff
@@ -50,4 +50,22 @@ def test_card_expiry_date
   def test_card_type
     assert_include @tester::CARD_TYPES, @tester.card_type
   end
+
+  def test_account_number
+    assert_instance_of String, @tester.account_number
+    assert_match(/\d{8}/, @tester.account_number(min_digits: 8, max_digits: 8))
+    assert_match(/\d{12}/, @tester.account_number(min_digits: 12, max_digits: 12))
```

Comment:
I think we always have to test whole string.
```suggestion
    assert_match(/\A\d{8}\z/, @tester.account_number(min_digits: 8, max_digits: 8))
    assert_match(/\A\d{12}\z/, @tester.account_number(min_digits: 12, max_digits: 12))
```

Reply:
Thank you. I've incorporated this change.